jest.config.js

```diff
@@ -5,5 +5,6 @@ module.exports = {
   setupFilesAfterEnv: ["<rootDir>/setupTests.js"],
   moduleNameMapper: {
     "\\.(css|scss)$": "<rootDir>/__mocks__/styleMock.js",
+    "^dist(.*)": "<rootDir>/dist/$1",
```

Comment:
we need to tell jest how to handle the webpack `dist` alias used in a Button import